src/lib/annotations/Annotator.js

```diff
@@ -835,6 +834,26 @@ class Annotator extends EventEmitter {
         return this.currentAnnotationMode === mode;
     }
 
+    /**
+     * Scrolls specified annotation into view
+     *
+     * @private
+     * @param {Object} threadID - annotation threadID for thread that should scroll into view
+     * @return {void}
+     */
+    scrollToAnnotation(threadID) {
+        if (!threadID) {
+            return;
+        }
+
+        Object.values(this.threads).forEach((pageThreads) => {
```

Comment:
I don't remember, do we have a polyfill for `Object.values()`? IE does not support it

Reply:
shoot. swapping it for object.keys

Reply:
Fixed that and others here https://github.com/box/box-content-preview/pull/371